tests/README.md

```diff
@@ -41,13 +41,42 @@ Integration tests require a running Kubevirt cluster.  Once you have a running
 Kubevirt cluster, you can use the `-master` and the `-kubeconfig` flags to
 point the tests to the cluster.
 
+## Running networking tests for outside connectivity
+
+When running the tests with no internet connection,
+some networking tests ,that test outside connectivity, might fail,
+and you might want to skip them.
+For that some additional flags are needed to be passed.
+In addition, if you'd like to test outside connectivity
+using different addresses than the default
+(`8.8.8.8`, `2001:db8:1::1` and `google.com`), you can achive that with the 
+designated flags as well.
+
+For each method detailed below, there is note about the needed flags
+to the outside connectivity tests and how to pass them.
+
 ## Run them on an arbitrary KubeVirt installation
 
 ```
 cd tests # from the git repo root folder
 go test -kubeconfig=path/to/my/config -config=default-config.json
 ```
 
+>**outside connectivity tests:** The tests will run by default.
+>To skip the outside connectivity tests add
+>`ginkgo.skip=\[outside_connectivity\]` To your go command.
+>To change the IPV4, IPV6 or DNS used for outside connectivity tests,
+>add `conn_check_ipv4_address`,
+>`conn_check_ipv6_address` or `conn_check_dns` to your go command,
+>with the desired value.
+>For example:
+>```
+>go test -kubeconfig=$KUBECONFIG -config=default-config.json
+>-conn-check-ipv4-address=8.8.4.4 > -conn-check-ipv6-address=2620:119:35::35
```

Comment:
what is this `>` ? typo i believe?

Reply:
It is a style. Might not be a successful one. I thought that since it is more of a note it should be like fade out. 
Do you think I should remove it?

Reply:
```
go test -kubeconfig=$KUBECONFIG -config=default-config.json
-conn-check-ipv4-address=8.8.4.4 > -conn-check-ipv6-address=2620:119:35::35
-conn_check_dns=amazon.com
```
This is how it looks on the markdown
the command is not right, the `>` redirects the output usually

maybe do it something like
```
go test -kubeconfig=$KUBECONFIG -config=default-config.json \
-conn-check-ipv4-address=8.8.4.4  \
-conn-check-ipv6-address=2620:119:35::35 \
-conn_check_dns=amazon.com
```

Reply:
What you pointed is my mistake. I thought you meant the > at the beginning of the line.
I will remove it.Thanks.